decidim-surveys/app/forms/decidim/surveys/survey_answer_form.rb

```diff
@@ -24,6 +26,10 @@ def map_model(model)
       def survey
         @survey ||= Survey.where(feature: current_feature).first
       end
+
+      def body_not_blank
+        errors.add("body", :blank) if body.all?(&:blank?)
```

Comment:
The body is an array because the user can choose multiple answers like: ["Answer A", "Answer B"]. I'm using the same field to store single answer questions as well so if the user doesn't fill in the answer it sends [""] to the server.